Lib/test/support/__init__.py

```diff
@@ -619,6 +620,12 @@ def sortdict(dict):
     withcommas = ", ".join(reprpairs)
     return "{%s}" % withcommas
 
+def run_code(code: str) -> dict[str, object]:
+    """Run a piece of code after dedenting it, and return its global namespace."""
+    ns = {}
+    exec(textwrap.dedent(code), ns)
+    return ns
```

Comment:
PEP 8 nit:

```suggestion


def run_code(code: str) -> dict[str, object]:
    """Run a piece of code after dedenting it, and return its global namespace."""
    ns = {}
    exec(textwrap.dedent(code), ns)
    return ns

```

Reply:
Sure. The file is terribly inconsistent about this already :)

Reply:
> The file is terribly inconsistent about this already

Doesn't mean we have to make the problem any worse ;)